code/addons/highlight/README.md

```diff
@@ -4,61 +4,149 @@ Storybook addon allows for highlighting specific DOM nodes within your story.
 
```

Comment:
Many of the suggestions made here also apply to `highlight.md`, as the content is near identical.